src/Microsoft.Fast.Components.FluentUI/Components/DateTime/FluentTimePicker.razor.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.Fast.Components.FluentUI.Utilities;
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+public partial class FluentTimePicker : FluentInputBase<DateTime?>
+{
+    /// <summary />
+    protected override string? StyleValue => new StyleBuilder()
+        .AddStyle(Style)
+        .Build();
+
+    /// <summary>
+    /// Gets or sets the design of this input.
+    /// </summary>
+    [Parameter]
+    public virtual TextFieldAppearance Appearance { get; set; } = TextFieldAppearance.Outline;
```

Comment:
Do we want to have a separate enum for this? Values are already available in `Appearance` enum. We can check if valid value was supplied in OnParameterSet(Async). Using 'just' Appearance keeps us consistent with web-component side.

Reply:
The **Appearance** enum contains many other values not available in the [text-field](https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/text-field/text-field.vscode.definition.json) web components.

I find it confusing to allow the developer to select a value that has no result or triggers an exception. In my opinion, the other "Text Field" components should be modified to use this reduced enumeration.

Reply:
Ok, maybe I'm clinging to much to the Fluent web components implementation here and now is a good time to diverge a bit.